packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -86,6 +87,7 @@ const _ActionList = ({ children, surfaceLevel = 2, testID }: ActionListProps): J
   } = useDropdown();
 
   const { theme } = useTheme();
+  const { isInBottomSheet } = useBottomSheetContext();
```

Comment:
Nope won't effect bundle, I've kept that in mind thus made this context providers independent of each other ensuring no direct dependency. 

This will only include few bytes for the context bundle size only.

Reply:
Nice, I think after #1132 is merged (likely before this PR) we can add the size-limit checks to ensure the code splitting is working fine, [example](https://github.com/razorpay/blade/pull/1132/files#diff-029ab5f2d1d465ecca3d6cecec05caeb7f2e6492a940253d13bb4b8c0abe1d17R247-R260)